packages/x-components/src/components/modals/base-events-modal.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <BaseModal
+    ref="baseModalEl"
```

Comment:
```suggestion
		v-bind="$attrs"
    ref="baseModalEl"
```
Move v-bind to first place to avoid warning:
https://v3-migration.vuejs.org/breaking-changes/v-bind.html